source/_integrations/roborock.markdown

```diff
@@ -17,10 +17,30 @@ ha_platforms:
 ha_integration_type: integration
 ---
 
-The Roborock integration allows you to control your [Roborock](https://us.roborock.com/pages/robot-vacuum-cleaner) vacuum while continuing to use the Roborock app.
+The Roborock integration allows you to control your [Roborock](https://us.roborock.com/pages/robot-vacuum-cleaner) vacuum while using the Roborock app.
 
-In contrast to [Xiaomi Miio](/integrations/xiaomi_miio/) integration, this integration provides more complete support but requires cloud access to set up and device configuration using the Roborock app.
+This integration requires a cloud connection to set up the device, but it communicates with the device entirely over your home network.
 
 Once you log in with your Roborock account, the integration will automatically discover your Roborock devices and get the needed information to communicate locally with them. Please ensure your Home Assistant instance can communicate with the local IP of your device. We recommend setting a static IP for your Roborock Vacuum to help prevent future issues.
 
 {% include integrations/config_flow.md %}
+
+## FAQ
+
+### Can I use the Mi home app with this integration?
+No. This integration requires information from your Roborock app to setup and uses Roborock's protocols to communicate with your device. You must have your vacuum synced to the Roborock app.
+
+### Can I block internet access for this device?
+As of right now - no. When the vacuum is disconnected from the internet, it will attempt to disconnect itself from wifi and reconnect itself until it can reach the Roborock servers. We are looking for the best way to handle this/ what can be blocked while still allowing the vacuum to function. 
```

Comment:
``` suggestion
As of right now - no. When the vacuum is disconnected from the internet, it will attempt to disconnect itself from Wi-Fi and reconnect itself until it can reach the Roborock servers. We are looking for the best way to handle this and see what can be blocked while still allowing the vacuum to function. 
```